Bottom: 0926a4bf52b8e126ca0aa65bb38b16ed340e1515
Top:    b2be28c058bcea1f57039dd4966c64ac692e6bfa
Author: Sean Hefty <sean.hefty@intel.com>
Date:   2012-06-11 14:10:47 -0700

Refresh of rs-man

---

diff --git a/man/rsocket.7 b/man/rsocket.7
index b26bab7..c6a2ad1 100644
--- a/man/rsocket.7
+++ b/man/rsocket.7
@@ -44,11 +44,11 @@ O_NONBLOCK
 In addition to standard socket options, rsockets supports options
 specific to RDMA devices and protocols.  These options are accessible
 through rsetsockopt using SOL_RDMA option level.
-.P
+.TP
 RDMA_SQSIZE - Integer size of the underlying send queue.
-.P
+.TP
 RDMA_RQSIZE - Integer size of the underlying receive queue.
-.P
+.TP
 RDMA_INLINE - Integer size of inline data.
 .P
 Note that rsockets fd's cannot be passed into non-rsocket calls.  For
